rmw_fastrtps_shared_cpp/package.xml

```diff
@@ -9,9 +9,11 @@
   <author>Ricardo González</author>
 
   <buildtool_depend>ament_cmake_ros</buildtool_depend>
+  <buildtool_depend>rosidl_cmake</buildtool_depend>
   <buildtool_depend>fastrtps_cmake_module</buildtool_depend>
 
   <buildtool_export_depend>ament_cmake</buildtool_export_depend>
+  <buildtool_export_depend>rosidl_cmake</buildtool_export_depend>
```

Comment:
Can you clarify why these two dependencies were added?

Reply:
my bad, it should be exported here in rmw_dds_common. But the package.xml is missing this `buildrool_export_depend`
https://github.com/ros2/rmw_dds_common/blob/master/rmw_dds_common/package.xml
https://github.com/ros2/rmw_dds_common/blob/master/rmw_dds_common/CMakeLists.txt#L25